backend/src/main/scala/com/softwaremill/bootzooka/api/Routes.scala

```diff
@@ -28,20 +46,12 @@ trait Routes extends UsersRoutes
     } & handleRejections(rejectionHandler)
   }
 
-  val routes =
+  def base(route: Route) =
```

Comment:
I've splitted it for testing, it was to big to my taste ;) Anyway not sure how to implement it other way and still be able to test it.